packages/nimble-components/src/rich-text/specs/README.md

```diff
@@ -200,6 +381,69 @@ _CSS Classes and CSS Custom Properties that affect the component_
 
 _Note_: This initial component design serves as a starting point for implementation, and it may undergo changes once the visual design is completed.
 
```

Comment:
It'd be helpful to make it super explicit which elements are non-visible configuration elements and which are visible UI elements. You allude to this in some of the UI elements but adding a note to each will make each of their purposes even more clear.

Reply:
Added